cmd/capk-manager/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
very basic manager for capk. It's copy pasted and slightly modified from the aws manager. You can skip this file, it's uninteresting.